osf_tests/test_preprint_summary.py

```diff
@@ -19,11 +19,26 @@ def preprint_provider():
 def preprint(preprint_provider):
     return PreprintFactory._build(PreprintService, provider=preprint_provider)
 
+
+@pytest.fixture()
+def right_before_my_birthday():
+    return {'preprint_dates': datetime.datetime(1991, 9, 25, 0, tzinfo=pytz.utc).replace(hour=23, minute=59, second=59),
+            'run_date': timezone.now()
```

Comment:
Is there a reason that this isn't an absolute date? Does something not work if it's not relative?

Reply:
I thought the idea was the dictionary would keep the current date if it failed again we'd know exactly when it failed. Now I'm unsure why "run_date" is included at all.

Reply:
`preprint_date` is the input for preprint.date_created. `run_date` is the input for `PreprintSummary().get_events`. Does that make sense?

Reply:
If not, drop by and we can chat.

Reply:
Makes sense.